ddtrace/context.py

```diff
@@ -20,25 +20,31 @@ class Context(object):
 
     This data structure is thread-safe.
     """
-    def __init__(self, trace_id=None, span_id=None, sampled=True):
+    def __init__(self, trace_id=None, span_id=None, sampled=True, sampling_priority=None):
         """
         Initialize a new thread-safe ``Context``.
 
         :param int trace_id: trace_id of parent span
         :param int span_id: span_id of parent span
         """
         self._trace = []
-        self._sampled = sampled
         self._finished_spans = 0
         self._current_span = None
         self._lock = threading.Lock()
-        self._parent_span_id = span_id
+
         self._parent_trace_id = trace_id
+        self._parent_span_id = span_id
+        self._sampled = sampled
+        self._sampling_priority = sampling_priority
+
+    def get_context_attributes(self):
```

Comment:
@LotharSee as we've discussed, we're hiding this API with the Injector/Extractor API so that users generates a `Context` using something more similar to OpenTracing.